_config.yml

```diff
@@ -33,6 +33,10 @@ search_tokenizer_separator: /[\s/]+/
 # Enable or disable heading anchors
 heading_anchors: true
 
+# Set the depth of displayed navigation links
+nav_top_show: 1    # levels always displayed
```

Comment:
What is the default behavior if this is unset (or nil)?

Reply:
It was hiding grandchildren. I've now adjusted a condition so that they are shown when `nav_active_show` is unset/nil, so the behaviour should be as in 0.2.7.

Reply:
I see there's now a conflict in nav.html. I'll try to resolve it...

Reply:
After merging 0.2.9 into the fork, I've resolved the conflict in nav.html, and tested the behaviour.
There's a minor issue: setting both `nav_top_show: 2` and `nav_active_show: 2` displays also the grandchildren.
It's non-trivial to fix that - partly due to limitations of conditions in Liquid - so I suggest to live with it for this release. Otherwise all the settings in the config work as expected (also when left unset).